src/main/java/com/ibm/etcd/client/EtcdClient.java

```diff
@@ -175,7 +174,7 @@ public Builder withDefaultTimeout(long value, TimeUnit unit) {
         }
         
         public Builder withPlainText() {
-            chanBuilder.usePlaintext(true);
+            chanBuilder.usePlaintext();
```

Comment:
Actually lets leave the deprecated (boolean arg) version of this one for now. It's a grpc-java method, not guava like the other deprecation updates, and the no-arg method was added in a very recent version IIRC. So this might prevent using with older versions of grpc-java which would otherwise be possible.